bittensor/_neuron/text/core_server/__init__.py

```diff
@@ -638,3 +629,27 @@ def time_check():
         except Exception as e:
             self.prometheus_counters.labels("blacklisted").inc()
             return True
+
+    def get_neuron(self):
+        if self.subtensor.network == 'finney':
+            nn = self.subtensor.get_neuron_for_pubkey_and_subnet(self.wallet.hotkey.ss58_address, netuid = self.config.netuid)
+        elif self.subtensor.network == 'nakamoto':
+            nn = self.subtensor.neuron_for_pubkey(self.wallet.hotkey.ss58_address)
+        return nn
+
+    def get_n(self):
```

Comment:
@isabella618033 what do you think about having more semantic in this function name??

From my point of view `get_n` should expose something more specific

Reply:
Renamed get_n to get_neuron_num.